website/docs/providers/change_notifier_provider.mdx

```diff
@@ -22,7 +23,7 @@ Prefer using [NotifierProvider] instead.
 Consider using `ChangeNotifierProvider` only if you are absolutely certain
 that you want mutable state.
 :::
-
+const immutabilityTitle = yaml.loadAll(await fs.readFile('/docs/concepts/immutability.mdx'))[0].title;
```

Comment:
@rrousselGit, would you mind helping me optimize this? I'm trying to programmatically get the title of the About Immutability page (see just a few lines down where the title is needed on this page). I'm in unfamiliar territory, and I haven't come across an example in the repo for what I'm trying to do.

Reply:
What are you trying to do exactly here?

Reply:
The intention was to ensure this part of the codebase would update automatically if the title of the About Immutability page ever changed again. Would you prefer the title to be hard-coded for the sake of simplicity?

Reply:
That's how I've done it so far.   
It's an interesting idea, but that's out of the scope of this PR I'd say.

We could make a separate PR to update links to automatically pick-up the title. That's not unique to this link.   
If we want to do this, we'd probably want to make a reusable component for this.

But that's a separate discussion.